library/common/config/config.cc

```diff
@@ -330,25 +329,7 @@ const char* config_template = R"(
           route_config:
             name: api_router
             virtual_hosts:
-            - name: h2_raw
-              include_attempt_count_in_response: true
-              virtual_clusters: *virtual_clusters
-              domains: *h2_raw_domains
-              routes:
-#{custom_routes}
-              - match: { prefix: "/" }
-                request_headers_to_remove:
-                - x-forwarded-proto
-                - x-envoy-mobile-cluster
-                route:
-                  cluster: base_h2
-                  timeout: 0s
-                  retry_policy:
-                    per_try_idle_timeout: *per_try_idle_timeout
-                    retry_back_off:
-                      base_interval: 0.25s
-                      max_interval: 60s
-            - name: primary
+            - name: api
```

Comment:
Should there be a comment here explaining why there needs to be a single virtual host, and that it must be named `api`?

Reply:
It does not need to named `api` anymore after we merged https://github.com/envoyproxy/envoy-mobile/pull/2548. Let me leave some comment with regard to the number of clusters.

Reply:
added a comment

Reply:
A single `R"()"` is now split in two with a C++ comment in between. Do these get concatenated?

I was thinking adding a YAML comment (`# comment...`) directly in the YAML string which doesn't have this risk, but I just don't know how C++ `R`-strings work here.